crypto/objects/objects.pl

```diff
@@ -166,14 +166,20 @@ sub expand
 	return $string;
 	}
 
+# Avoid mucking up the FIPS checksums, which look at include/openssl/obj_mac.h
+my @fips_exclude_list = qw(brotli zstd);
+my %fips_exclude = map { $_ => undef } @fips_exclude_list;
+
 foreach (sort { $a <=> $b } keys %ordern)
 	{
 	$Cname=$ordern{$_};
 	print "\n";
+	print "#ifndef FIPS_MODULE\n" if exists $fips_exclude{$sn{$Cname}};
```

Comment:
I had meant to remove `brotli` since it was not longer considered "special" (like zlib). It's now just a regular option to be used with `enable-` or `no-`.

Reply:
It may not even be needed for 3.1...